OneSignalSDK/onesignal/src/main/java/com/onesignal/OSSessionManager.java

```diff
@@ -159,9 +167,9 @@ void onSessionStarted() {
             //session already set
             return;
         setLastNotificationsId();
-        if (notificationIds.length() > 0) {
-            session = Session.INDIRECT;
-            OneSignal.Log(OneSignal.LOG_LEVEL.DEBUG, "Session indirect with notificationId: " + notificationIds);
+        if (hasIndirectNotifications()) {
+            setSession(Session.INDIRECT);
+            OneSignal.Log(OneSignal.LOG_LEVEL.DEBUG, "Session indirect with directNotificationId: " + indirectNotificationIds);
```

Comment:
Technically we only use this `directNotificationId` in the `DIRECT` case
And we only `indirectNotificationIds` in the `INDIRECT` case

I named them so it would make more sense as to what type of session relies on them

Reply:
OHH you meant the log haha got it